ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -1381,3 +1381,40 @@ periodics:
     - configMap:
         name: image-mirror-mappings
       name: config
+- agent: kubernetes
+  cluster: app.ci
+  cron: '@yearly'
+  decorate: true
+  labels:
+    ci.openshift.io/area: image-mirroring-on-demand
+    ci.openshift.io/role: image-mirroring
+  name: periodic-image-mirroring-on-demand
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        failures=0
+        for mapping in $IMAGE_PATHS; do
+          image_name_tag=${mapping##*/}
+          cmd="oc image mirror --keep-manifest-list ${mapping} registry.ci.openshift.org/ci/${image_name_tag} --skip-multiple-scopes"
```

Comment:
@hongkailiu not sure what is missing from:
```
oc image mirror --keep-manifest-list ${mapping} registry.ci.openshift.org/ci/${image_name_tag} --skip-multiple-scopes
```
` ${mapping}` is the source. e.g `quay.io/example/image1:latest`
and the target is
`registry.ci.openshift.org/ci/${image_name_tag}` e.g ` registry.ci.openshift.org/ci/image1:latest`

Reply:
@hongkailiu our images are mapped in `core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci`; 1h period is too long for us.
`Our user can still trigger the existing job via gangway.` - how?

Reply:
@rnetser I discussed yesterday with @hongkailiu your case. Hongkai is one of the most experienced people in our team in this subject.

His points about risk for our platform are valid. Our plan to avoid that risk and enable you going forward would be:
1. We drop this PR completely
2. You can run via gangway API aka REST API  `mapping_supplemental_ci_images_ci` job directly. The job is intelligent enough to compare images that do not need any sync so it takes usually around 1-3 mins to execute the job. Of course downside is that you will sync every image which is not synced but for now we can live with that. We will observe and mitigate if needed. So to sum up this point: include all the images you potentially need in `mapping_supplemental_ci_images_ci` and run it.
3. The part that worried me is: `If 1h was too long for the user, manual trigger the job would not avoid it completely. Because it still relies on the test-image-distributor to sync the image from app.ci to build0n which could be 24h delay.` That is a case even today. To counter it, after syncing using REST API, you should reach quay.io and build0x registry (the registry that your test currently runs in) and compare shas. It would we wise to proceed only if they are the same, which means you need to have a reasonable timeout if it takes more then couple of minutes (10? 15? The decision is yours.).

In the long-term we eventually will move to quay.io with our registry so there is a potential for 3) to disappear, but right now we can't make any promises. Our strong feeling is that in most of the situations syncing takes no more than several minutes, but the reality is that in rare cases it can be 24h and as Hongkai explained to me it depends on some k8s magic.

Reply:
You can consult the job here to see the exec times: https://prow.ci.openshift.org/?job=periodic-image-mirroring-supplemental-ci-images